metaprotocol/assetlist.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema":"../assetlist.schema.json",
+    "chain_name":"LumenX",
+    "assets":[
+       {
+          "description":"The native token of Lumen Network",
+          "denom_units":[
+             {
+                "denom":"ulumen",
+                "exponent":0
+             },
+             {
+                "denom":"lumen",
+                "exponent":6
+             }
+          ],
+          "base":"ulumen",
+          "name":"LUMEN",
+          "display":"lumen",
+          "symbol":"LUMEN",
+          "logo_URIs":{
+             "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/metaprotocol/images/lumen.png",
+             "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/metaprotocol/images/lumen.svg"
```

Comment:
an svg is not required, but if you have it, can you please find a proper svg? The one uploaded is simply a non-vector image saved with a .svg extension, has a white background where it should be transparent.

Reply:
Thanks for the kind review. I will delete unnecessary svg file.